sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1279,7 +1279,7 @@ object RapidsConf {
       "values are ALL: print everything, NONE: print nothing, NOT_ON_GPU: print only parts of " +
       "a query that did not go on the GPU")
     .stringConf
-    .createWithDefault("NONE")
+    .createWithDefault("NOT_ON_GPU")
```

Comment:
As an engineer on the spark-rapids team I have the explain=ALL constantly. But as a user, I might be less inclined to see even explain=NOT_ON_GPU output without asking for it. 

I think we either should either keep it off, but maybe advertise it in the https://github.com/NVIDIA/spark-rapids/blob/9defb4cc08ae3853d546ac7e29744067b4c18bcd/sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala#L84. Alternatively keep the default but advertise how it can be turned off

Reply:
I think it was decided based on customer feedback that change to NOT_ON_GPU and advertise how to turn it off.  If this becomes an issues with other customers we could turn off again next release.

Reply:
In this case, I suggest to advertise it more prominently in the plugin init phase logPluginMode

Reply:
Hi @gerashegalov, do you mean to print a clear log to tell what spark.rapids.sql.explain is in logPluginMode function?

Reply:
Is it a good idea to advertise it in the docs `get-started/getting-started.md`?

Reply:
> Hi @gerashegalov, do you mean to print a clear log to tell what spark.rapids.sql.explain is in logPluginMode function?

yes, I mean we could print something like:

spark.rapids.sql.explain is set to the default value NOT_ON_GPU to print operations not executed on GPU. Set to ALL to print the complete query plan. Set to NONE to suppress the diagnostics about the query placement on the GPU.